packages/client-core/src/world/reducers/scenes/reducers.ts

```diff
@@ -1,26 +1,26 @@
-import { SCENES_FETCHED_ERROR, SCENES_FETCHED_SUCCESS, SET_CURRENT_SCENE } from '../../reducers/actions'
```

Comment:
Please delete this file instead of commenting out if it's not going to be used now